Gemfile

```diff
@@ -108,6 +108,7 @@ source "https://rails-assets.org" do
   gem "rails-assets-bootstrap-datepicker",     "~>1.4.0"
   gem "rails-assets-bootstrap-hover-dropdown", "~>2.0.11"
   gem "rails-assets-bootstrap-select",         "~>1.7.3"
+  gem 'rails-assets-kubernetes-topology-graph', '=0.0.16'
```

Comment:
In a1d63f2, Brandond made the quoting consistent, could you change to double quotes please?